infrastructure/laa-check-client-qualifies-uat/ingress-dashboard.yaml

```diff
@@ -692,8 +692,8 @@ data:
           {
             "current": {
               "selected": false,
-              "text": "main-laa-estimate-eligibility",
-              "value": "main-laa-estimate-eligibility"
+              "text": "main-laa-check-client-qualifies",
```

Comment:
Ooh, so this one is a bit misleading. In the old namespace, the release name of main UAT used to be `main-la-estimate-eligibility`, but at a certain point it got changed to `check-client-qualifies-laa-estimate-eligibility` but we never updated this config file to match. In the new namespace that's also the release name of main UAT, so the correct value of this should be `check-client-qualifies-laa-estimate-eligibility`